### PR TITLE
Display AM/PM for ko-KR culture

### PR DESCRIPTION
**Pull request by EgorBo**
Fixes https://github.com/mono/mono/issues/11317
(I guess I need to update culture-info-tables.h via make but it doesn't work on Windows, I have to switch to macOS).

### CODE REVIEW

File: mcs/class/corlib/Test/System/DateTimeTest.cs [hunk 1]
```diff
@@ -2667,5 +2667,18 @@ public void ToUniversalTime_TimeZoneOffsetShouldNotOverflow ()
...
+
+		[Test] // https://github.com/mono/mono/issues/11317
+		public void DateTimeKoCulture ()
+		{
+			foreach (var culture in new [] { new CultureInfo ("ko"), new CultureInfo ("ko-KP"), new CultureInfo ("ko-KR") })
+			{
+				var dateTimeAm = new DateTime (2018, 1, 1, 11, 0, 0);
+				var dateTimePm = new DateTime (2018, 1, 1, 13, 0, 0);
+
+				Assert.AreEqual ("오전 11:00:00", dateTimeAm.ToString ("T", culture.DateTimeFormat));
```
Comment (akoeplinger) [inline mcs/class/corlib/Test/System/DateTimeTest.cs]: We usually avoid having the Unicode symbols in the .cs and instead encode them via the C# syntax, i.e. the above would be `"\uC624\uC804 11:00:00"`

File: tools/locale-builder/Driver.cs [hunk 2]
```diff
@@ -1144,6 +1144,11 @@ void Import (XmlDocument doc, CultureInfoEntry ci)
 				case "en-NZ":
 					df.AMDesignator = "a.m.";
 					break;
+				case "ko":
+				case "ko-KP":
+				case "ko-KR":
+					df.AMDesignator = "오전";
```
Comment (akoeplinger) [inline tools/locale-builder/Driver.cs]: same here

Comment (marek-safar) [reply tools/locale-builder/Driver.cs]: I am fine with this, it reads better ;-)

**Comment by luhenry** [comment]
@marek-safar can you please re-review.

**Comment by marek-safar** [comment]
@monojenkins backport 2018-10

**Comment by marek-safar** [comment]
@monojenkins backport 2018-12

**Comment by marek-safar** [comment]
@monojenkins backport 2018-08

**Pull request merged**